test/test_datasets.py

```diff
@@ -1504,29 +1503,30 @@ class MovingMNISTTestCase(datasets_utils.DatasetTestCase):
 
     ADDITIONAL_CONFIGS = combinations_grid(split=(None, "train", "test"), split_ratio=(10, 1, 19))
 
+    _NUM_FRAMES = 20
+
     def inject_fake_data(self, tmpdir, config):
         base_folder = os.path.join(tmpdir, self.DATASET_CLASS.__name__)
         os.makedirs(base_folder, exist_ok=True)
-        num_samples = 20
+        num_samples = 5
         data = np.concatenate(
             [
                 np.zeros((config["split_ratio"], num_samples, 64, 64)),
-                np.ones((20 - config["split_ratio"], num_samples, 64, 64)),
+                np.ones((self._N_FRAMES - config["split_ratio"], num_samples, 64, 64)),
```

Comment:
This also needs to be updated

```suggestion
                np.ones((self._NUM_FRAMES - config["split_ratio"], num_samples, 64, 64)),
```